package.json

```diff
@@ -4,7 +4,8 @@
   "description": "show programming language based on the repos of github's account",
   "main": "index.js",
   "scripts": {
-    "start": "nodemon lib/app.js | gulp sass",
+    "start": "nodemon lib/app.js | gulp watch",
+    "build": "nodemon lib/app.js | gulp",
```

Comment:
when do we use these scripts? I mean the `gulp watch` and `gulp`

Reply:
`start` for development, you can try running on locally and change some SCSS on `base.scss`, try see log terminal and `build` just once compiled the SCSS

Reply:
I see. So are you saying we should use `npm start` for prod environment and `npm build` for development?

Reply:
npm run start for development, npm run build for production

Reply:
ok good suggestion, will try on my local first. but can you change the script to `npm start-dev` instead of `npm build`? for better readability. thanks!

Reply:
@wildan3105 updated

Reply:
ah ya can you help me to make this issue hacktoberfast? interest article https://hacktoberfest.digitalocean.com/hacktoberfest-update @wildan3105 👌🏻😎

Reply:
sure!